Argh, CI has failed:

```
gometalinter -j2 --config "/home/travis/gopath/src/github.com/crucibuild/sdk-agent-go/gometalinter.json" ./...
WARNING: deadline exceeded by linter safesql (try increasing --deadline)
make: *** [check] Error 2
The command "make ci" exited with 2.
Done. Your build exited with 1.
```

--- a/Makefile
+++ b/Makefile
@@ -8,7 +8,7 @@ dependencies:
 check: dependencies
 	go get -u github.com/alecthomas/gometalinter
 	gometalinter --install --update
-	gometalinter -j2 --config "$(CURDIR)/gometalinter.json" ./...
+	gometalinter -j2 --deadline 60s --config "$(CURDIR)/gometalinter.json" ./...
 
 coverage:
 	go get golang.org/x/tools/cmd/cover

--- a/gometalinter.json
+++ b/gometalinter.json
@@ -12,7 +12,6 @@
         "interfacer",
         "megacheck",
         "misspell",
-        "safesql",
         "structcheck",
         "unparam",
         "varcheck"
@@ -20,6 +19,7 @@
     "Unused": [
         "lll",
         "dupl",
-        "goimports"
+        "goimports",
+        "safesql"
     ]
 }

